.gitattributes

```diff
@@ -1,11 +1,13 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
 * text=auto
 
+MacOS_users_read_this_first.txt eol=lf
```

Comment:
I like the approach, will review it in depth sometimes when I have the free
time for it.

Bernát GÁBOR

On Wed, Feb 3, 2016 at 9:34 AM, Dan Van Atta <notifications@github.com>
wrote:

> In .gitattributes
> <https://github.com/triplea-game/triplea/pull/426#discussion_r51691525>:
>
> > @@ -1,11 +1,13 @@
> >  # Set the default behavior, in case people don't have core.autocrlf set.
> >  * text=auto
> >
> > +MacOS_users_read_this_first.txt eol=lf
>
> Moving the exceptions towards the top, so they stand out more (I was
> hoping order would matter and rules would take precedence, not the case)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/triplea-game/triplea/pull/426/files#r51691525>.
>